civictechprojects/models.py

```diff
@@ -40,7 +40,7 @@ class Project(models.Model):
     project_links = models.CharField(max_length=5000, blank=True)
 
     def __str__(self):
-        return str(self.id) + ':' + self.project_name
+        return str(self.id) + ':' + str(self.project_name)
```

Comment:
It technically does work, I cast it to string because of a warning from my IDE that self.project_name was a CharField, rather than a string. I don't think it's important to change.